api/base/utils.py

```diff
@@ -29,6 +30,8 @@
 
 UPDATE_METHODS = ['PUT', 'PATCH']
 
+hashids = Hashids(alphabet='abcdefghijklmnopqrstuvwxyz')
```

Comment:
We should pass a salt so that the hashids aren't decodable by anyone except the app: https://github.com/davidaurelio/hashids-python#using-a-custom-salt .

The salt should be configurable in `api/base/settings/local.py`, and it should be made required here: https://github.com/CenterForOpenScience/osf.io/blob/826c1cfe3825af8e2f156d235084758c3785306c/api/base/settings/__init__.py#L27-L28